### DISCUSSION TITLE
[ FF7 ] Game crashes when choosing save file

### DISCUSSION THREAD

**Discussion by frylock567** [julianxhokaxhiu/FFNx#178] [General]
Happened after I updated to latest 7th Heaven release
[crashfiles.zip](https://github.com/julianxhokaxhiu/FFNx/files/5563993/crashfiles.zip)

**Comment by julianxhokaxhiu** [comment]
![image](https://user-images.githubusercontent.com/1237070/99678288-f03f7780-2a7a-11eb-9deb-8527d101b375.png)

**Comment by frylock567** [comment]
Wait where am i supposed to post crash info then. Sorry, I just followed the directions of the error message I got.

**Comment by julianxhokaxhiu** [comment]
On the relative 7th Heaven support channels. No worries :)
